tools/data/textrecog/totaltext_converter.py

```diff
@@ -345,30 +352,21 @@ def load_img_info(files):
 def parse_args():
     parser = argparse.ArgumentParser(
         description='Convert totaltext annotations to COCO format')
-    parser.add_argument('root_path', help='totaltext root path')
-    parser.add_argument('-o', '--out-dir', help='output path')
+    parser.add_argument('root_path', help='Totaltext root path')
     parser.add_argument(
-        '--split-list',
-        nargs='+',
-        help='a list of splits. e.g., "--split_list training test"')
-
-    parser.add_argument(
-        '--nproc', default=1, type=int, help='number of process')
+        '--nproc', default=1, type=int, help='Number of process')
     args = parser.parse_args()
     return args
 
 
 def main():
     args = parse_args()
     root_path = args.root_path
-    out_dir = args.out_dir if args.out_dir else root_path
-    mmcv.mkdir_or_exist(out_dir)
-
     img_dir = osp.join(root_path, 'imgs')
     gt_dir = osp.join(root_path, 'annotations')
 
     set_name = {}
-    for split in args.split_list:
+    for split in ['training', 'test']:
         set_name.update({split: 'instances_' + split + '.json'})
```

Comment:
The annotation files generated in the code are `train_label.txt` and `test_label.txt`, but the print values here are `instance_training.json` and `nstance_test.json`
should we use 
```python
set_name.update({split: split+'_label'+'.txt'})
```